src/gui/shareuserline.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>400</width>
-    <height>310</height>
+    <width>419</width>
+    <height>544</height>
```

Comment:
If a size change is necessary, I would say it would be better to choose round numbers IMO